modules/world/map/map_interface.cpp

```diff
@@ -105,7 +97,81 @@ bool MapInterface::FindNearestLanes(const Point2d &point,
   return true;
 }
 
-bool MapInterface::isInLane(const modules::geometry::Point2d &point, LaneId id) const
+LanePtr MapInterface::FindLane(const Point2d& point) const {
+  
+  LanePtr lane;
+  std::vector<LanePtr> nearest_lanes;
+  
+  // TODO: parameter (20) auslagern
+  if(!FindNearestLanes(point, 20, nearest_lanes, false)) {
```

Comment:
Use ParameterServer straight away.. can define default value there

Reply:
I've created an issue for that #156